plugins/modules/swim_workflow_manager.py

```diff
@@ -150,6 +153,13 @@
               schedule_origin:
                 description: ScheduleOrigin query parameter. Originator of this call (optional).
                 type: str
+          cco_image_details:
+            description: Details of the image needed to import from the cisco.com
+            type: dict
+            suboptions:
+              Image_name:
```

Comment:
the variable name should be 'image_name'.

Reply:
updated the code